src/game-container/game-gfx-container/GfxContainer.svelte

```diff
@@ -1,56 +1,161 @@
 <script>
-    import utilFunctions from '../../helpers/utilFunctions'
-    import GameBlockRow from './GameBlockRow.svelte'
+    import util from '../../helpers/utilFunctions'
+    import initOverworld from '../../game/initOverworld'
+    import createCharInstance from '../../game/createCharInstance'
 
-    console.log(utilFunctions)
+    export let classList
 
-    utilFunctions.docReady(function() {
+    let gameChar = {}
+    let frontCtx;
+    let backCtx;
 
-        const gridWidth     = document.getElementById("game-gfx-body").offsetWidth
-        const gridHeight    = document.getElementById("game-gfx-body").offsetHeight
-        const gridDimension = [ 48, 32 ]
-        const blockDimensions = gridWidth / 48
+    function startGame() {
+        const charName      =  util.getInputVal('name')
+        const charGender    =  util.getInputVal('gender')
+        const charClass     =  util.getInputVal('class')
 
-        console.log(gridWidth / 24)
-        console.log(gridHeight / 16)
+        // The setTimeouts setup is not definitive and might change later
+        setTimeout( () => {
+            document.getElementById('intro-screen').remove()
+
+        }, 50 )
+
+        setTimeout( () => {
+            backCtx     = initOverworld.initCanvas(0)      
+            frontCtx    = initOverworld.initCanvas(1)     
```

Comment:
Weni hoe serieus je dit allemaal wilt nemen hoor maar wij hebben hier iig conventie dat we alles uitschrijven en geen afkortingen etc. gebruiken zodat andere developers en jij id toekomst geen enkele kans hebt om iets niet te begrijpen
Ctx -> Context gaat het dan over, ga er iig vanuit dat het context is, dus zou hier wel opmerking krijgen van ey bro maak er ff backContext van